application/modules/setup/sql/016_1.4.4.sql

```diff
@@ -10,6 +10,16 @@ SET setting_value = 'InvoicePlane - paid'
 WHERE setting_key = 'pdf_invoice_template_paid' AND
       setting_value = 'default-paid';
 
+UPDATE ip_settings
+SET setting_value = 'InvoicePlane - unpaid'
```

Comment:
don't edit old .sql files to add your fields, just create new ones

Reply:
Okay